clients/unshaded/src/main/java/tachyon/client/file/options/SetStateOptions.java

```diff
@@ -45,6 +46,17 @@ public Builder setPinned(boolean pinned) {
     }
 
     /**
+     * @param ttl the ttl(time to live) value (in milliseconds) to use; it specifies how long should
```

Comment:
if you would like to change the description, update the `CreateOptions#setTTL` description as well